### DISCUSSION TITLE
New/Browse Disk Images

### DISCUSSION THREAD

**Discussion by Pontiac76** [86Box/86Box#3788] [Q&A]
I'm having a problem trying to create a new floppy disk image in 86Box.  Part of the problem, I think, is that I lost my NAS where I had some floppy disk and CDRom images kicking around, and when I try to go to Media> Floppy> New Image, the VM just locks up trying to browse to a probably dead directory as far as my OS is concerned.

Where in the hosting registry (Win10 Pro) is 86Box defaulting to, or how does it retain where to look?

**Comment by cold-brewed** [comment]
This generally will be whatever Qt defaults to for your OS. Here are some possibilities:

* directory that contains the binary
* directory that contains your 86box config
* root directory (macOS)
* last directory used in an open dialog

To fix this, you can either:

* Manually edit your `86box.cfg` and remove old media images, or

* Go into 86Box application preferences (note: not the same as system settings) and put a check by "Select media images from working directory" to force it to always default to the directory that contains your config file

**Comment by Pontiac76** [comment]
There's two parts to this.

I went through the hosts registry to hunt down any network drive I had, and nuked them (The NAS is dead and written off, so I don't need the drive letters anyways).  I still had a problem after I was done ONCE, but went to bed, and the next morning, I was fine, except, for the following.

I think I actually found what the problem is, and I consider this a bug.

I have a RAM drive on my machine, and it's typically empty except for a Temp directory and something I had put on there to test networking on a physical 486 with a NIC.  If I do this process fast enough, if I select what path I want to map as an ISO, if I select the root of the RAM drive (IE: R:\) the UI locks up nice and tight, but, I can hear the background stuff going on, like the memory check, and POST successful beep.  But nothing kicks over once I'm out of the dialogue.

But, if I repeat the process (As in kill the process, restart 86box), and select something like R:\Temp\ or R:\Grrr\ (Valid paths) for a CDRom, I get a return no problem, and can continue on.

And after I test by selecting R:\Temp or R:\Grrr, and go select R:\, the UI locks up again, nice and tight.

The host CPU looks to be busy as well with the process, locked or not locked, sitting at a DOS prompt.
